nbflow/extractor.py

```diff
@@ -57,7 +57,7 @@ def get_dependencies(self, dirnames):
                 sources = [self.resolve_path(filename, x) for x in params['__depends__']]
 
                 targets = params['__dest__']
-                if not hasattr(targets, '__iter__'):
+                if isinstance(targets, str):
```

Comment:
Strings in Python 3 have an `__iter__` method, which was causing some rather interesting erroneous targets. But now I see that you're right, this version breaks on `None` instead. Need to check for that.